core/templates/domain/story/story-summary.model.ts

```diff
@@ -112,6 +124,30 @@ export class StorySummary {
     return this._classroomUrlFragment;
   }
 
+  getPublishedChaptersCount(): number | undefined {
+    return this._publishedChaptersCount;
+  }
+
+  getTotalChaptersCount(): number | undefined {
+    return this._totalChaptersCount;
+  }
+
+  getUpcomingChaptersCount(): number | undefined {
+    return this._upcomingChaptersCount;
+  }
+
+  getOverdueChaptersCount(): number | undefined {
+    return this._overdueChaptersCount;
+  }
+
+  getVisitedChapterTitles(): string[] | undefined {
+    return this._visitedChapterTitles;
+  }
+
+  getUpcomingChaptersExpectedDays(): number[] | undefined {
+    return this._upcomingChaptersExpectedDays;
+  }
+
```

Comment:
@seanlip which one do we prefer? Why?
  - `get<VariableName>` 
  - getter functions (`get <variableName>`)

Reply:
No strong preference from my end. I think the former is more standard in our codebase since this codebase has existed for a while.

Reply:
All similar functions in the file have been written using the first format, so I have also written the same way.